File: app/src/main/java/com/example/mad2/landingpage/LandingPageVM.java
package com.example.mad2.landingpage;

import android.os.Bundle;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class LandingPageVM extends ViewModel {
    private MutableLiveData<LandingPageModel> landingPageModel = new MutableLiveData<>(new LandingPageModel());
    private Bundle navArguments;

    public MutableLiveData<LandingPageModel> getLandingPageModel() {
        return landingPageModel;
    }

    public void setLandingPageModel(MutableLiveData<LandingPageModel> landingPageModel) {
        this.landingPageModel = landingPageModel;
    }



    public Bundle getNavArguments() {
        return navArguments;
    }

    public void setNavArguments(Bundle navArguments) {
        this.navArguments = navArguments;
    }
}

File: jia_hui_s_application/app/src/main/java/com/jiahuisapplication/app/modules/landingpage/data/viewmodel/LandingPageVM.kt
package com.jiahuisapplication.app.modules.landingpage.`data`.viewmodel

import android.os.Bundle
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel
import com.jiahuisapplication.app.modules.landingpage.`data`.model.LandingPageModel
import org.koin.core.KoinComponent

class LandingPageVM : ViewModel(), KoinComponent {
  val landingPageModel: MutableLiveData<LandingPageModel> = MutableLiveData(LandingPageModel())

  var navArguments: Bundle? = null
}
